joomla/joomla-cms
# Change the listing of each language file to a folder tag

Pull Request for Issue #14200 .

### Summary of Changes
Replaces the list of `<filename>en.GB.foo.ini</filename>` with a single `<folder>/</folder>` tag.

### Testing Instructions
* Download an language pack of your choice and edit both install.xml files within it with this change.
* Install the pack, check that it worked (eg all files are copied as expected)
* Uninstall the pack and verify the files are removed

### Documentation Changes Required
Documentation for creating a language pack should be adjusted.
https://docs.joomla.org/J3.x:Making_a_Language_Pack_for_Joomla#an_install.xml
